examples/naming-conventions/Cargo.toml

```diff
@@ -1,19 +0,0 @@
-[package]
-name = "uniffi-example-naming-conventions"
```

Comment:
Perhaps controversially, I have deleted the `naming-conventions` example. Reason being that, when I added these tests, I discovered that the `sprites` example was failing in Kotlin due to not being updated for the naming conventions. So I think we can safely check the behaviour of those conventions through the existing examples.